### PR TITLE
Initial documentation regarding the writing of tests

### PR DESCRIPTION
**Pull request by ewjoachim**
Do we agree on the methodology ? If so, let's write tests :)

### CODE REVIEW

**Comment by ewjoachim** [comment]
Well, I was about to start writing tests and then (procrastination hit and) I realized I wasn't really clear how I wanted to write them. So I started writing ideas down, and soon enough I was there. I know that a lot of things regarding documentation are probably going to move so I'm not putting too much thought into what goes where for now, but I wanted to make sure I wasn't starting on a path alone.

**Comment by ewjoachim** [comment]
BTW, thanks for the review !

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Everything that's not unit tested should be integation tested, so we may aim for a  coverage rate that's highest as possible, but it's interesting to look at the unit coverage on itself to make sure we didn't forget important unit tests.
+
+## Tools
+
+`pytest` and `coverage`. Install with:
+
+```console
+$ pip install -e ".[test]"
+```
```
Comment (k4nar) [inline tests/README.md]: Maybe we could add a line about launching the tests?

Comment (ewjoachim) [reply tests/README.md]: As soon as we have tests, yeah !

**Pull request merged**